.github/workflows/est-ds-realm-separate-test.yml

```diff
@@ -83,14 +83,33 @@ jobs:
 
           docker exec ca pki nss-cert-import --cert estSSLServer.crt sslserver
 
-          docker exec ca pk12util -d /root/.dogtag/nssdb -o $SHARED/est_server.p12 -n sslserver -W Secret.123
+          docker exec ca pki pkcs12-cert-import sslserver --pkcs12-file $SHARED/est_server.p12 --pkcs12-password Secret.123
 
       - name: Add CA EST user
         run: |
           docker exec ca pki -n caadmin ca-group-add "EST RA Agents"
           docker exec ca pki -n caadmin ca-user-add \
               est-ra-1 --fullName "EST RA 1" --password Secret.est
           docker exec ca pki -n caadmin ca-group-member-add "EST RA Agents" est-ra-1
+
+      - name: Create CA EST user certificate end store top p12
+        run: |
+          docker exec ca pki nss-cert-request --csr estUser.csr \
+              --ext /usr/share/pki/server/certs/admin.conf --subject 'UID=estUser'
+
+          docker exec ca pki \
+              -n caadmin \
+              ca-cert-issue \
+              --csr-file estUser.csr \
+              --profile caUserCert \
+              --output-file estUser.crt
+
+          docker exec ca pki nss-cert-import --cert estUser.crt estUser
+          
+          CERT_ID=$(docker exec ca pki nss-cert-show estUser |sed -n "s/^\s*Serial Number:\s*\(\S*\)$/\1/p")
+          docker exec ca pki -n caadmin ca-user-cert-add est-ra-1 --serial $CERT_ID
```

Comment:
I think we can use `pki ca-user-cert-add --input` so no need to parse the serial number.

https://github.com/dogtagpki/pki/wiki/PKI-User-Certificate-CLI